From eb681fa09092862b826d18e07b68678535e6dcc9 Mon Sep 17 00:00:00 2001
From: Jackeagle <jackeagle102@gmail.com>
Date: Tue, 26 Mar 2019 16:38:25 +0100
Subject: [PATCH 3/3] Revert "prebuilt/etc: Create an additional Bliss OTA
 Package directory /data/"

This reverts commit cc4484b4f2a9def3e24be8152dc5df3bfdd6a252.
---
 prebuilt/common/etc/init.bliss.rc | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/prebuilt/common/etc/init.bliss.rc b/prebuilt/common/etc/init.bliss.rc
index 464d04c50..b2c8e3dca 100644
--- a/prebuilt/common/etc/init.bliss.rc
+++ b/prebuilt/common/etc/init.bliss.rc
@@ -20,9 +20,6 @@ on init
 
 on post-fs-data
     mkdir /data/.ssh 0750 root shell
-    
-    # Create an additional OTA package directory that unlike /data/bliss_updates
-    mkdir /data/bliss_updates 0770 system cache
 
     # Run sysinit
     start sysinit
-- 
2.17.1

